runtime-jvm/main/src/main/scala/Lib2.scala

```diff
@@ -42,26 +42,18 @@ object Lib2 {
   }
 
   def builtin1(decompiled: Term, n: Name, f: NumericUnaryOp): Lambda = {
-    val body = new Computation(null) {
-      def apply(r: R, rec: Lambda, top: StackPtr,
-                stackU: Array[U], x1: U, x0: U,
-                stackB: Array[B], x1b: B, x0b: B): U = {
-        r.boxed = null
-        f(x0)
-      }
+    val body: Computation = (r,rec,top,stackU,x1,x0,stackB,x1b,x0b) => {
+      r.boxed = null
+      f(x0)
```

Comment:
I'm fine with having all the args or just the needed args for the SAM lambda, but kinda want to pick one style.